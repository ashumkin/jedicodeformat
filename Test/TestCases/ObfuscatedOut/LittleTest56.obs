unit littletest56;interface implementation const foo=true;{$IF not Foo}

  warrawak


{$ELSE}const bar=3;{$IFEND}end.