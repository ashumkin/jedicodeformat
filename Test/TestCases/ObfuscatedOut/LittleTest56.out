unit littletest56;

interface

implementation

const
  foo = True;
{$IF not Foo}

  warrawak


{$ELSE}

const
  bar = 3;{$IFEND}

end.
